modules/android/pages/kotlin.adoc

```diff
@@ -320,7 +320,7 @@ In Use::
 ----
 
 val docModDate = db.documentChangeFlow("1001", null)
-    .map { it.collection.getDocument(it.documentID)?.getString("lastModified") }
+    .map { it.collection.getDocument(it.collection)?.getString("lastModified") }
```

Comment:
I'm pretty sure that this change is a mistake.... I think it should stay "documentID"